doc/nrf/known_issues.rst

```diff
@@ -2302,12 +2339,12 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
```

Comment:
Some new known issues copied from nrfxlib changelog "bugfix" section (FYI @rugeGerritsen ): 

rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-18261: Wrongly uses non-zero randomness for the first advertising event
   The controller used non-zero randomness even after calling :c:func:`sdc_hci_cmd_vs_set_adv_randomness` with a valid ``adv_handle`` parameter

rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-18358: Scanning may get hardfaults
   The controller would end up in the HardFault handler after receiving an invalid response to a scan request.

rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-18411: LE Connection complete event has wrong peer_address_type if resolved address
   The ``Peer_Address_Type`` parameter in the ``LE Connection Complete`` event was incorrectly set to 2 or 3 in case the connection was established to a device which address was resolved.

rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0

DRGN-18420: Periodic advertiser may get NULL pointer dereference
   The controller could dereference a NULL pointer when starting a periodic advertiser.

rst-class:: v2-2-0

DRGN-18586: Assert when starting Periodic Advertisement Sync Transfer while Periodic Advertising is not enabled
   When initiating Periodic Advertising Sync Transfer (PAST) as advertiser, the controller might assert if the periodic advertising train is not running.

rst-class:: v2-2-0

DRGN-18655:  Wrongly set the address if calling :c:func:`bt_ctlr_set_public_addr` before :c:func:`bt_enable
   `bt_ctlr_set_public_addr()` accesses uninitialized memory if called before `bt_enable()`

Reply:
The 18655 entry is missing description, @laxiLang , and the description is required.

Reply:
> Some new known issues

@laxiLang , "some" suggests the list is not complete. Are these all known issues that need to be added?

Reply:
If @rugeGerritsen has no further comments, "some" --> whole list (w.r.t posted in nrfxlib sdc changelog)